io/net/htx_nic_devices.py.data/htx_nic_devices.yaml

```diff
@@ -0,0 +1,9 @@
+mdt_file: 'net.mdt'
+host_ip: x.xx.xxx.xx
```

Comment:
We need one peer ip(test interface ip) for executing ssh commands on peer, and also public IP's of both Host & Peer we need to update those in /etc/hosts & bpt files.